Makefile

```diff
@@ -144,15 +148,14 @@ build/$(API_JAR): $(API_SOURCES)
 
 build/$(CONVERTER_JAR): $(CONVERTER_SOURCES) src/converter/MANIFEST.MF
 	mkdir -p build/converter
+	cp -r $(COMMON_RES_DIR)/* build/converter
+	#cp -r $(JAVA_RES_DIR)/* build/converter
 	$(JAVAC) -source 7 -target 7 -d build/converter $(CONVERTER_SOURCES)
 	$(JAR) cfm $@ src/converter/MANIFEST.MF -C build/converter .
 	$(RM) -r build/converter
 
-%.class.h: %.class
-	hexdump -v -e '1/1 "%u,"' $^ > $@
-
 %.class: %.java
-	$(JAVAC) -g:none -source $(JAVAC_RELEASE_VERSION) -target $(JAVAC_RELEASE_VERSION) $(*D)/*.java
+	$(JAVAC) -g:none -d $(CPP_RES_DIR) -source $(JAVAC_RELEASE_VERSION) -target $(JAVAC_RELEASE_VERSION) $(*D)/*.java
```

Comment:
Here is a tiny out-of-scope change: instead of put class.h to the same folder, I'm putting .class files to res/cpp.